EXAMPLE.md

```diff
@@ -0,0 +1,82 @@
+
+---
+RFC: <PR number>
```

Comment:
I like this, can we attach a reference to the README so the users know they can copy this?